internal/attestation/measurements/measurements.go

```diff
@@ -181,10 +181,20 @@ func (m *M) SetEnforced(enforced []uint32) error {
 type Measurement struct {
 	// Expected measurement value.
 	Expected [32]byte `json:"expected" yaml:"expected"`
-	// WarnOnly if set to true, a mismatching measurement will only result in a warning.
-	WarnOnly bool `json:"warnOnly" yaml:"warnOnly"`
+	// MeasurementValidationOpt indicates how measurement mismatches should be handled.
+	MeasurementValidationOpt MeasurementValidationOption `json:"warnOnly" yaml:"warnOnly"`
```

Comment:
```suggestion
	// ValidationOpt indicates how measurement mismatches should be handled.
	ValidationOpt MeasurementValidationOption `json:"warnOnly" yaml:"warnOnly"`
```
Why not go for the shorter `ValidationOpt` when the struct is already named `Measurement`?